.travis.yml

```diff
@@ -1,109 +1,26 @@
-#
-# Travis Setup
-#
+language: php
 
-# use ubuntu trusty for newer version of nodejs, used for JS testing
-dist: trusty
+php:
+  - '7.2'
+  - '7.3'
+  - '7.4snapshot'
 
 # faster builds on new travis setup not using sudo
-# temporary disable, see https://github.com/travis-ci/travis-ci/issues/6842
-#sudo: false
-sudo: required
-group: edge
-
-# build only on master branches
-# commented as this prevents people from running builds on their forks:
-# https://github.com/yiisoft/yii2/commit/bd87be990fa238c6d5e326d0a171f38d02dc253a
-#branches:
-#  only:
-#    - master
-#    - 3.0
-
-
-#
-# Test Matrix
-#
-
-language: php
-
-env:
-  global:
-    - DEFAULT_COMPOSER_FLAGS="--prefer-dist --no-interaction --no-progress --optimize-autoloader"
-    - TASK_TESTS_PHP=1
-    - TASK_TESTS_COVERAGE=0
+sudo: false
 
 # cache vendor dirs
 cache:
   directories:
-    - vendor
     - $HOME/.composer/cache
-    - $HOME/.npm
-
-# try running against postgres 9.6
-addons:
-  code_climate:
-    repo_token: 2935307212620b0e2228ab67eadd92c9f5501ddb60549d0d86007a354d56915b
-
-matrix:
-  fast_finish: true
-  include:
-    - php: 7.3
-      env: DEFAULT_COMPOSER_FLAGS="$DEFAULT_COMPOSER_FLAGS --ignore-platform-reqs" # php-cs
-    - php: 7.2
-      env: TASK_TESTS_COVERAGE=1
-    - php: nightly
-
-  allow_failures:
-    - php: nightly
 
 install:
-  - |
-    if [[ $TASK_TESTS_COVERAGE != 1 ]]; then
-      # disable xdebug for performance reasons when code coverage is not needed
-      phpenv config-rm xdebug.ini || echo "xdebug is not installed"
```

Comment:
Without this one tests will be launched with XDebug and will run significantly slower. Let's have it as in https://github.com/yiisoft/yii-web/blob/master/.travis.yml

Reply:
Actually, whole config could be similar.